packages/twenty-server/src/engine/core-modules/file/controllers/file.controller.ts

```diff
@@ -44,6 +44,10 @@ export class FileController {
         workspaceId,
       );
 
+      fileStream.on('error', () => {
```

Comment:
I have noticed that local driver (at least) does not throw FileNotFound as the stream get piped. do we  have to change the API here of our file drivers or is there a way to make it work like the s3 driver?
@Weiko any idea?

Reply:
I would update the read function in local driver yes. I thought it was done though, my bad! @charlesBochet